src/server/game/Globals/ObjectMgr.h

```diff
@@ -1326,6 +1384,18 @@ class TC_GAME_API ObjectMgr
             if (itr == _pointOfInterestLocaleStore.end()) return nullptr;
             return &itr->second;
         }
+        PlayerChoiceLocale const* GetPlayerChoiceLocale(uint32 ChoiceID) const
+        {
+            PlayerChoiceLocaleContainer::const_iterator itr = _playerChoiceLocaleContainer.find(ChoiceID);
+            if (itr == _playerChoiceLocaleContainer.end()) return nullptr;
+            return &itr->second;
+        }
+        PlayerChoiceResponseLocale const* GetPlayerChoiceResponseLocale(uint32 ChoiceID, uint32 ResponseID) const
```

Comment:
move implentation out of class scope?

Reply:
Consistency, all other implentation are in class scope

Reply:
can be done just like 
```c
    return Trinity::Containers::MapGetValuePtr(_playerChoiceResponseLocaleContainer, std::make_pair(ChoiceID, ResponseID));

```

and same above